gst/tensor_ros_src/tensor_ros_src.cc

```diff
@@ -508,18 +525,21 @@ gst_tensor_ros_src_create (GstPushSrc * src, GstBuffer ** buffer)
   gsize size = rossrc->count * tensor_element_size[rossrc->datatype];
 
   /* get item from queue */
-  queue_item = g_async_queue_try_pop (rossrc->queue);
+  while (true) {
+    queue_item = g_async_queue_timeout_pop (rossrc->queue, G_USEC_PER_SEC / rossrc->freq_rate);
+    if (queue_item) {
+      GST_DEBUG_OBJECT (rossrc, "queue_item exists!!!\n");
+      break;
+    }
+    /* @todo Return EOF or error */
```

Comment:
You need two *s to let Doxygen understand.

```/*``` --> ```/**```

Reply:
I fixed it. Thanks.